corehq/apps/domain/views/releases.py

```diff
@@ -79,6 +87,49 @@ def post(self, request, *args, **kwargs):
             return self.get(request, *args, **kwargs)
 
 
+@method_decorator([toggles.RELEASE_BUILDS_PER_PROFILE.required_decorator()], name='dispatch')
+class ManageReleasesByAppProfile(BaseProjectSettingsView):
+    template_name = 'domain/manage_releases_by_app_profile.html'
+    urlname = 'manage_releases_by_app_profile'
+    page_title = ugettext_lazy("Manage Releases By App Profile")
+
+    @cached_property
+    def form(self):
+        return ManageReleasesByAppProfileForm(
+            self.request,
+            self.domain,
+            data=self.request.POST if self.request.method == "POST" else None,
+        )
+
+    def _get_initial_app_profile_details(self, version):
+        return [{}]
+
+    @property
+    def page_context(self):
+        app_names = {app.id: app.name for app in get_brief_apps_in_domain(self.domain, include_remote=True)}
+        query = LatestEnabledBuildProfiles.objects.order_by('version')
+        # ToDo: Filter by params in url
+        version = self.request.GET.get('version')
+        app_releases_by_app_profile = [release.to_json(app_names) for release in query]
+        return {
+            'manage_releases_by_app_profile_form': self.form,
+            'app_releases_by_app_profile': app_releases_by_app_profile,
+            'selected_build_details': ({'id': version, 'text': version} if version else None),
+            'initial_app_profile_details': self._get_initial_app_profile_details(version),
+        }
+
+    def post(self, request, *args, **kwargs):
+        if self.form.is_valid():
+            success, error_message = self.form.save()
```

Comment:
please ignore post method.
This view does not have a post view as of now, but this is how it would work in future.
The form would have a save method that would return a bool success and error_message if any. So this is removed in a future commit.